### DISCUSSION TITLE
MetaMask Unity SDK (Contract Interaction)

### DISCUSSION THREAD

**Discussion by YusufBesim** [MetaMask/snaps#1259] [Q&A] [answered]
Hello,

Firstly thanks for this great plugin.

I worked on plugin last 3 day. Tried all functions like sign, send Ether etc.
But can't find how to interact with specific contract.
I'm trying to mint NFT from the contract

Thank you.

**Comment by ritave** [comment, marked as answer]
Hello Yusuf, this is discussions concerning MetaMask Snaps functionality, not the SDK. You should consider going to MetaMask SDK Discord: https://discord.gg/mheVnqqGYC
